<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
  <meta http-equiv="Content-Style-Type" content="text/css">
  <title></title>
  <meta name="Generator" content="Cocoa HTML Writer">
  <meta name="CocoaVersion" content="1404.47">
  <style type="text/css">
body{ text-align:center; margin:0; background-image: url('http://i65.tinypic.com/25rf8dc.jpg'); background-color: #EFEFEF; background-attachment:fixed;}
div.column{width: 70%; max-width: 70%; margin: 0 auto; text-align: left; position: relative;}
div.header{margin: 0; background-color:teal; top: 0;
  z-index: 999; 
width: inherit;
 display: block; 
  position: fixed;}
div.headercontents{padding: 6pt; }

div.page{ margin: 60px 0 0 0; font-family: "Open sans", Verdana; background-color: #FFF; opacity: 0.8; }
div.page table{padding: 6pt;}

   .position{padding: 0; width:100%; margin: 20pt 0 20pt 0; }
   .education td.role{padding: 0 0 12pt;}
   .position a:hover, p a:hover{color: #32B2B2; transition: color 1s ease;}
   p a{color: #000}
   .date{font-family: "Open sans", Verdana; font-size: 14pt; color: black; font-style: italic; width: 20%; text-align: center; valign: top;}
   .title, .title a{font-family: "Open sans", Verdana; font-size: 20pt; color: black; padding: 0;}
   .place, .place a{font-family: "Open sans", Verdana; font-size: 14pt; color: black; padding: 0 0 12pt 0; line-height: 50%;}
   .project, .role{font-family: "Open sans", Verdana; font-size: 10pt; color: black; font-weight: bold;}
   .leftcol{font-family: "Open sans", Verdana; font-size: 10pt; color: black; font-style: italic; width: 13%;}
   .date2{font-family: "Open sans", Verdana; font-size: 10pt; color: black; width: 10%;}
   .position p{font-family: "Open Sans", Verdana; font-size: 10pt; font-weight: normal; text-align: justify;}
 

 h1{font-family: "Open sans", Verdana; font-size: 24pt; color: white; font-weight: normal; line-height: 80%; padding: 2pt 0 2pt 10pt; margin: 0; text-decoration: none; font-style: italic; background: #32B2B2; /* For browsers that do not support gradients */
    background: -webkit-linear-gradient(left, #32B2B2, teal); /* For Safari 5.1 to 6.0 */ac
    background: -o-linear-gradient(to right, #32B2B2, teal); /* For Opera 11.1 to 12.0 */
    background: -moz-linear-gradient(to right, #32B2B2, teal); /* For Firefox 3.6 to 15 */
    background: linear-gradient(to right, #32B2B2, teal); /* Standard syntax */ border-style:solid; border-width: 1px 0; border-color: teal; opacity: 1;}
p.pub{padding: 6pt;}

p.up{display:block; text-align:center;}
.up a{display:block; margin: auto; font-weight: bold;  font-size: 24pt; text-decoration: none; background-color: #000; border-radius: 50px; color:#FFF; width: 50px; height: 50px;}
.up a:hover{background-color: #EFEFEF; transition: background-color 1s ease;}

</style>


</head>

<body>

<div class="column">
<div class="header">
<div class="headercontents">
<A href="mailto:AMJEve-CV@yahoo.co.uk">Alex Eve</a>

<div class="menu">



</div>
</div>


</div>
<div class="page">
<h1><a name="current">Current Position</a></h1>

<table class="position">
<tr><td class="date">Jul 2016 - Present</td><td class="title" colspan="3" ><a href="http://biosciences.exeter.ac.uk/staff/index.php?web_id=Alex_Eve">Associate Research Fellow</a></td></tr>
<tr><td></td><td class="place" colspan="3">Ingham Group, <a href="http://www.exeter.ac.uk/livingsystems/">Living Systems Institute</a>, University of Exeter</td></tr>
<tr><td></td><td class="leftcol">Key projects:</td><td class="project" colspan="2">"Evolution of the Hedgehog signalling pathway." <br /></td></tr>
<tr><td></td><td></td><td colspan="2" class="role">
<p>Hedgehog signalling is cruical for the development of most animals although the mechanism has diverged between vertebrate and invertebrates.</p><p>I am developing a <i>Nematostella vectensis</i> model for the mechanistic and functional study of Hedgehog pathway components. I am characterising the Hedgehog pathway and loss of function phenotype in <i>Nematotella</i> and generating <i>Nematostella</i> genetic mutants using CRISPR/Cas9 to study the role of Cos2/Kif7, Fused, and the requirement for ciliary-mediated signal transduction.</p>
</td></tr>
<tr><td></td><td class="leftcol">Additional roles:</td><td class="date2">Nov 2016</td><td class="role">Visiting scientist: The Francis Crick Institute</td></tr>
<tr><td></td><td></td><td colspan="2"><p></p></td></tr>

<tr><td></td><td></td><td class="date2">Oct 2016</td><td class="role">Collaborator: A*STAR Institute of Molecular and Cell Biology</td></tr>
<tr><td></td><td></td><td colspan="2"><p></p></td></tr></table>

<h1><a name="research">Research Experience</a></h1>

<table class="position">
<tr><td class="date">Jan 2017 - Mar 2017</td><td class="title" colspan="3" >Guest Research Fellow</td></tr>
<tr><td></td><td class="place" colspan="3"><a href="http://www.sars.no/research/RentzschGrp.php">Rentzsch Group, Sars Centre for Molecular Marine Biology</a></td></tr>
<tr><td></td><td class="leftcol">Key projects:</td><td class="project" colspan="2">"Evolution of the Hedgehog signalling pathway."</td></tr>
<tr><td></td><td></td><td colspan="2">
<p>Thanks to a <A href="">Company of Biologists</a>, Development <a href="#grants">Travel Fellowship</a> I was able to visit the lab of Fabian Rentzsch for two months. Here, I was trained to culture <i>Nematostella vectensis</i> as a developmental model organism and performed some preliminary experiments investigating the role of Hedgehog signalling in <i>Nematostella</i> development.</p><p>See some of my research images from the trip <a href="images.html">here</a>.</p>
</td></tr>
</table>

<table class="position">
<tr><td class="date">Jan 2017 - Mar 2017</td><td class="title" colspan="3">Postdoctoral Research Assistant</td></tr>
<tr><td></td><td class="place" colspan="3"><A href="https://www.crick.ac.uk/research/a-z-researchers/researchers-p-s/jim-smith/">Smith Group, The Francis Crick Institute</a>, Mill Hill Laboratory</td></tr>
<tr><td></td><td class="leftcol" rowpsan=2>Key projects:</td><td class="project" colspan="2">"Comparison of <i>tmem88a</i> mutants and morpholino knockdown phenotypes."</td></tr>
<tr><td></td><td></td><td colspan="2">
<p>Tmem88a is believed to be a negative regulator of the Wnt signalling pathway, required for cardiogenesis and primitive haematopoiesis, although studies have obtained inconsistent results using morpholino oligonucleotides.</p>
<p>I generated an 8 bp deletion in the coding region of the tmem88a locus using TALENs, and established a viable homozygous <i>tmem88a</i>&Delta;8 mutant line. Although <i>tmem88a</i>&Delta;8 mutants have reduced expression of some key haematopoietic genes, differentiation of primitive haematopoiesis is unaffected. I found that expression of the <i>tmem88a</i> paralogue <i>tmem88b</i> is not significantly changed in <i>tmem88a</i>&Delta;8 mutants and injection of the <i>tmem88a</i> splice-blocking morpholino oligonucleotide into <i>tmem88a</i>&Delta;8 mutants causes the reduction of erythrocytes observed in morphants. This suggested that there is a partial, but inessential, requirement for <i>tmem88a</i> during haematopoiesis and that nonspecific off target effects of morpholino injection exacerbates this phenotype in <i>tmem88a</i> knockdown embryos. (<a href="http://dx.doi.org/10.1371/journal.pone.0172227">Eve et al. 2017</a>)</p>
</td></tr>
<tr><td></td><td></td><td class="project" colspan="2">"Investigating the role of <i>lamc3</i> during zebrafish development."</td></tr>
<tr><td></td><td></td><td colspan="2">
<p>Laminins are a major component of the basement membrane. A number of zebrafish mutants exist for different laminin chains, however there has been no functional study of laminin &gamma;3.</p>
<p><i>In situ</i> and transcriptome data suggests that <i>lamc3</i> is expressed in the vasculature of developing zebrafish. Knockdown using morpholino oligonucleotides prevents neural migration and development of the lymphatic system, although with possible off target effects. Future work will aim to generate a <i>lamc3</i> genetic mutant for further functional study.
</td></tr>
</table>

<table class="position">
<tr><td class="date">Jan 2017 - Mar 2017</td><td class="title" colspan="3">PhD Student</td></tr>
<tr><td></td><td class="place" colspan="3"><a href="https://www.crick.ac.uk/research/a-z-researchers/researchers-p-s/jim-smith/">Smith Group, The Francis Crick Institute</a>, Mill Hill Laboratory</td></tr>
<tr><td></td><td class="leftcol">Thesis:</td><td class="project" colspan="2">"Identification and functional analyses of novel genes in haemovasculogensis".</td></tr>
<tr><td></td><td></td><td colspan="2">
<p>I used previously generated RNA-sequencing data to functionally characterise novel genes with roles in haemovascular development (<A href="http://dx.doi.org/10.1016/j.mod.2012.10.002">Cannon et al. 2013</a>). Genes enriched in endothelial cells were knocked down using morpholino oligonucleotides and phenotypes were characterised in transgenic zebrafish lines. Two genes of interest, <i>tmem88a</i> and <i>lamc3</i>, were selected for further study.</p>

</td></tr>
</table>

<table class="position">
<tr><td class="date">Jan 2017 - Mar 2017</td><td class="title" colspan="3">Undergraduate Student</td></tr>
<tr><td></td><td class="place" colspan="3"><a href="http://www.port.ac.uk/institute-of-biomedical-and-biomolecular-science/molecular-biophysics/garry-scarlett/">Scarlett Group</a>, Department of Biophysics, University of Portsmouth</td></tr>
<tr><td></td><td class="leftcol">Dissertation:</td><td class="project" colspan="2">"Characterisation of a novel <i>Xenopus</i> double stranded RNA-binding protein."</td></tr>
<tr><td></td><td></td><td colspan="2">
<p>I used RT-PCR to attempt characterise the temporal expression pattern of a novel dsRNA-binding protein in <i>Xenopus laevis</i>.</p>
</td></tr>
</table>


<h1><a name="education">Education</a></h1>

<table class="position">
<tr><td class="date" rowpsan="2">Sep 2011 - Sep 2015</td><td class="title">University College London</td></tr>
<tr><td></td><td class="place">PhD Cell and Developmental Biology</td></tr>
<tr><td class="date" rowpsan="2">Sep 2008 - Jul 2011</td><td class="title">University of Portsmouth</td></tr>
<tr><td></td><td class="place">BSc (Hons) Biochemistry</td></tr>
<tr><td class="date" rowpsan="3">Sep 2006 - Jul 2008</td><td class="title">SEEVIC College</td></tr>
<tr><td></td><td class="place">A-level: Law (A), Biology (A), Chemistry (B)</td>
<tr><td></td><td class="place">AS level: Psychology (A)</td></tr>
<tr><td class="date" rowpsan="4">Sep 2001 - Jul 2006</td><td class="title">King Edmund School</td></tr>
<tr><td></td><td class="place">AVCE: ICT (A)</td>
<tr><td></td><td class="place">AS level: Science for Public Understanding (B)</td></tr>
<tr><td></td><td class="place">9 GCSEs (A*-B)</td></tr>
</table>

<h1><a name="publications">Grants and awards</a></h1>



<h1><a name="publications">Publications</a></h1>

<p class="pub"><b>Eve AMJ</b>, Place ES, and Smith JC (2017) Comparison of Zebrafish <i>tmem88a</i> mutant and morpholino knockdown phenotypes. <i>PLoS ONE</i> 12(2): e0172227. <A href="http://dx.doi.org/10.1371/journal.pone.0172227">DOI:10.1371/journal.pone.0172227</a></p>
<p class="pub">Cannon JE, Place ES, <b>Eve AMJ</b>, Bradshaw CR, Sesay A, Morrell NW, and Smith JC (2013). Global analysis of the haematopoietic and endothelial transcriptome during zebrafish development. <i>Mech. Dev.</i> 130, 122–31. <A href="http://dx.doi.org/10.1016/j.mod.2012.10.002">DOI: 10.1016/j.mod.2012.10.002</a></p>
<div class="center"><p class="up"><a class="top" href="#top">&#8613;</a></p></div>

<div style="height:500px;">&nbsp;</div>
</div>
</div>


</body>
</html>
